internal/zinc-persist/src/main/scala/xsbt/api/CompanionsFormat.scala

```diff
@@ -0,0 +1,16 @@
+package xsbt.api
+
+import xsbti.api._
+import sbinary._
+
+object CompanionsFormat extends Format[Companions] {
```

Comment:
I want to rely less on Java serialization.. eventually I want to turn all these into JSON so we can cross machine boundaries safely.